### DISCUSSION TITLE
Welcome to XQuickEnergy Discussions!

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#241] [Announcements/公告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pidanpro** [comment]
只开一个古树保护，保护了几十个然后就一直显示访问被拒绝，是保护的太快被制裁了吗？

**Comment by dayuanzong** [comment]
感谢分享
